app/workers/web/push_notification_worker.rb

```diff
@@ -48,11 +60,38 @@ def perform(subscription_id, notification_id)
         end
       end
     end
-  rescue ActiveRecord::RecordNotFound
-    true
   end
 
-  private
+  def perform_standard_request
```

Comment:
Rather than duplicating the entire request method, it'd be better to have like a `prepare_headers` and `prepare_payload` methods, leaving the rest the same.